app/controllers/api/v1/events_controller.rb

```diff
@@ -25,54 +23,54 @@ def group_events
     end
 
     def add_movie
-      event = Event.find(params[:event_id])
+      event_1 = Event.find(params[:event_id])
       if event.update(imdb_id: params[:movie])
-        render json: event, status: 201
+        render json: event_1, status: 201
       else
-        render json: { errors: event.errors }, status: 422
+        render json: { errors: event_1.errors }, status: 422
       end
     end
 
     def attending
-      event = Event.find(params[:event][:id])
-      current_user = User.find_by(access_token: params[:user][:access_token])
-      event.users << current_user
+      event.users << user
       render json: event.users
     end
 
     def not_attending
-      event = Event.find(params[:event][:id])
-      user = User.find_by(access_token: params[:user][:access_token])
       event.users.delete(user)
       render json: event.users
     end
 
     def add_rating
-      event = Event.find(params[:event][:id])
-      user = User.find_by(access_token: params[:user][:access_token])
-      rating = params[:event][:rating]
-      remove_rating(event, user) if user_has_already_rated(event, user)
-      rating_record = Rating.create
-      if rating_record.update(user_id: user.id, rating_score: rating, event_id: event.id)
-        render json: rating_record
-      else
-        render json: { errors: rating_record.errors }
-      end
+      rating = Rating.find_or_initialize_by(user_id: user.id, event_id: event.id)
+      rating.update(rating_score: params[:event][:rating])
+      render json: rating
     end
 
     def show_rating
-      event = Event.find(params[:id])
-      user = User.find(params[:user_id])
-      average = get_average_rating(event)
-      if rating = find_rating(event, user)
-        render json: { rating: rating, average: average }
-      else
-        render json: { rating: {rating_score: 0}, average: average }
-      end
+      event_1 = Event.find(params[:id])
```

Comment:
Yeah see my long comment up higher... @Owenhorner under your comment "I wont merge those variable names in"